apps/expo/src/app/muscleGroup/index.tsx

```diff
@@ -0,0 +1,219 @@
+import React, { useState } from 'react'
+import { Dimensions, Image, Text, View, StyleSheet } from 'react-native'
+import { router } from 'expo-router'
+import tailwind from '@/tooling/tailwind'
+import { Ionicons, MaterialCommunityIcons } from '@expo/vector-icons'
+import { SafeAreaView } from 'react-native-safe-area-context'
+import Body from 'react-native-body-highlighter'
+import ToggleSwitch from '~/components/toggle/Toggle'
+import FrontBackSwitch from '~/components/muscleGroup/FrontBackSwitch'
+
+export type muscleSelect =
+  | "abs"
+  | "adductors"
+  | "ankles"
+  | "biceps"
+  | "calves"
+  | "chest"
+  | "deltoids"
+  | "deltoids"
+  | "feet"
+  | "forearm"
+  | "gluteal"
+  | "hamstring"
+  | "hands"
+  | "hair"
+  | "head"
+  | "knees"
+  | "lower-back"
+  | "neck"
+  | "obliques"
+  | "quadriceps"
+  | "tibialis"
+  | "trapezius"
+  | "triceps"
+  | "upper-back";
+
+export default function NewWorkoutMuscleGroup() {
+  const { width: screenWidth, height: screenHeight } = Dimensions.get('window')
+  const vmin70 = Math.min(screenWidth, screenHeight) * 0.7
+
+  const [bodyPartSelected, setBodyPartSelected] = useState({/*
+    slug: "biceps",
+    intensity: 2,*/
+  }
+  )
+  const [isBackSideEnabled, setIsBackSideEnabled] = useState(false)
+  const [isMale, setIsMale] = useState(true)
```

Comment:
maybe we could modify the user schema so that it won't always default to `male` if they are a female? something to think about. women who use this app will always have to switch every time they open this up.

Reply:
add gender in user, and acquire through global context is the easiest way here.

Reply:
Agreed. Maybe we can ask the user on account setup if they have a preference. Could be my next issue. Marcos and I were talking about this last night (creating usernames and stuff)

Reply:
Do you intend to make a questionnaire upon user creation or just make the preferences in the account settings page,

Reply:
popup upon user creation. will discuss in meeting today